<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>weikang</author>
 <class>DemoWindow</class>
 <widget class="QMainWindow" name="DemoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>820</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>演示</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="main_layout" stretch="3,1">
      <item>
       <layout class="QGridLayout" name="view_layout" rowstretch="2,3" columnstretch="0,0">
        <item row="0" column="0">
         <widget class="QWidget" name="wd_left" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <widget class="QLabel" name="lb_left">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>cam_main</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="wd_right" native="true">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <widget class="QLabel" name="lb_right">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>cam_3rd</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWidget" name="wd_bottom" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="control_layout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dataset_dir">
            <property name="text">
             <string>tmp/data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_open_dir">
            <property name="toolTip">
             <string/>
            </property>
            <property name="statusTip">
             <string>打开数据集目录</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="btn_voice">
            <property name="text">
             <string>语音输入</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btn_arm_zero">
            <property name="text">
             <string>机械臂回零位</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>任务</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_tasks"/>
            </item>
            <item>
             <widget class="QPushButton" name="btn_run">
              <property name="text">
               <string>执行</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="msg">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="text">
           <string>realtime_msg</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPlainTextEdit" name="txt_log">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_clear">
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_setting">
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>打开设置页</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::PreferencesRole</enum>
   </property>
  </action>
  <action name="act_gripper">
   <property name="text">
    <string>夹爪开合</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
